src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,16 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
+
```

Comment:
Private constructor is needed to avoid possibility to create instance of class HibernateUtil
```suggestion
    private HibernateUtil() {
    }
```